rllib/algorithms/algorithm_config.py

```diff
@@ -3260,11 +3257,11 @@ def fault_tolerance(
                 setting is ignored.
             ignore_env_runner_failures: Whether to ignore any EnvRunner failures
```

Comment:
Ah, here it becomes clear. Maybe we refer to this section or maybe I overread it above.